components/LanguageButton.tsx

```diff
@@ -1,12 +1,24 @@
 import Link from 'next/link';
 import Button from './Button';
+import Icons from './Icons';
 
 function LanguageButton({ language }: { language: string }) {
+
+  let lan:string = language.toLowerCase();
```

Comment:
Also the code isn't formatted. Can you format all the code with prettier before pushing?